Update manage-virtual-instance.md (#105818)
Azure/azure-content: articles/sap/center-sap-solutions/manage-virtual-instance.md

:::image type="content" source="media/configure-virtual-instance/sap-resources.png" lightbox="media/configure-virtual-instance/sap-resources.png" alt-text="Screenshot of VIS resource in Azure portal, showing SAP resources pages in the sidebar menu for ASCS, App server, and Database instances.":::

## Default Instance Numbers

> If you've deployed an SAP system using Azure Center for SAP solutions, below shows the default values of instance numbers configured during deployment :

Reviewer: ```suggestion
If you've deployed an SAP system using Azure Center for SAP solutions, the following list shows the default values of instance numbers configured during deployment:
```

- Distributed Systems [HA and non-HA systems]

> 1. ASCS Instance Number - 00
> 2. ERS Instance Number - 01
> 3. DB Instance Number - 00
> 4. APP Instance Number - 00

Reviewer: ```suggestion
   - ASCS Instance Number - 00
   - ERS Instance Number - 01
   - DB Instance Number - 00
   - APP Instance Number - 00
```

- Single Server Systems 
1. ASCS Instance Number - 01
3. DB Instance Number - 00

> 4. APP Instance Number - 02

Reviewer: ```suggestion
   - ASCS Instance Number - 01
   - DB Instance Number - 00
   - APP Instance Number - 02
```

## Connect to SAP Application

> To connect to and manage SAP Application, you can use following credentials :

Reviewer: ```suggestion
To connect to and manage SAP Application, you can use the following credentials:
```

> 1. User : DDIC or RFC_USER or SAP*
> 2. Client ID : 000

Reviewer: ```suggestion
 - User : DDIC or RFC_USER or SAP*
 - Client ID : 000
```
